terraform/modules/ecs-user/iam.tf

```diff
@@ -6,6 +6,7 @@ resource "aws_iam_role" "user_host" {
 
 data "aws_iam_policy_document" "assume_role_ec2" {
   statement {
+    sid     = "AllowAssumeRole"
```

Comment:
I'm not sure why we need to name the SIDs from the security review, but I would assume it should give an indicator of why the policy is needed, so some of these names seem too generic.